src/bin/download_sysext.rs

```diff
@@ -342,7 +345,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     fs::create_dir_all(&temp_dir)?;
 
     // The default policy of reqwest Client supports max 10 attempts on HTTP redirect.
-    let client = Client::builder().redirect(Policy::default()).build()?;
+    let client = Client::builder().connect_timeout(Duration::from_secs(HTTP_CONN_TIMEOUT)).redirect(Policy::default()).build()?;
```

Comment:
Is there any way to add a timeout for reads? I only see https://docs.rs/reqwest/latest/reqwest/struct.ClientBuilder.html#method.timeout which is for the whole response. This is problematic because we would need a very large timeout for slow downloads, like one hour (But anyway still better than being stuck indefinitely like now).

Reply:
```suggestion
    let client = Client::builder().connect_timeout(Duration::from_secs(HTTP_CONN_TIMEOUT)).timeout(Duration::from_secs(DOWNLOAD_TIMEOUT)).redirect(Policy::default()).build()?;
```
with DOWNLOAD_TIMEOUT being 3600